concepticondata/conceptlists/Teit-1908-693.tsv

```diff
@@ -0,0 +1,694 @@
+Teit-1908-693-87	87	73	ear	1247	EAR	46
+Teit-1908-693-88	88	74	earlobe	18	EARLOBE	46
+Teit-1908-693-89	89	75	nose	1221	NOSE	46
+Teit-1908-693-90	90	76	nose ridge			46
+Teit-1908-693-91	91	77	nostril	1178	NOSTRIL	46
+Teit-1908-693-92	92	78	cheek	1730	CHEEK	46
+Teit-1908-693-93	93	79	back	1291	BACK	46
+Teit-1908-693-94	94	80	beard	678	BEARD	46
+Teit-1908-693-95	95	81	mouth	674	MOUTH	48
+Teit-1908-693-96	96	82	lip	478	LIP	48
+Teit-1908-693-97	97		all [lips]			48
+Teit-1908-693-98	98		upper [lip]			48
+Teit-1908-693-99	99	83	lower lip			48
+Teit-1908-693-100	100	84	lower jaw			48
+Teit-1908-693-101	101	85	tooth	1380	TOOTH	48
+Teit-1908-693-102	102	86	tongue	1205	TONGUE	48
+Teit-1908-693-103	103	87	saliva	1503	SALIVA	48
+Teit-1908-693-104	104	88	spittle			48
+Teit-1908-693-105	105	89	throat	1346	THROAT	48
+Teit-1908-693-106	106	90	chin	1510	CHIN	48
+Teit-1908-693-107	107	91	neck	1333	NECK	48
+Teit-1908-693-108	108	92	body	1480	BODY	48
+Teit-1908-693-109	109	93	shoulder more upper part	1482	SHOULDER	48
+Teit-1908-693-110	110	94	shoulder more lower part or point	172	SHOULDERBLADE	48
+Teit-1908-693-111	111	95	front of body			48
+Teit-1908-693-112	112	96	chest	1592	CHEST	48
+Teit-1908-693-113	113	97	breast of woman	1402	BREAST	48
+Teit-1908-693-114	114	98	belly	1251	BELLY	48
+Teit-1908-693-115	115	99	belly (more inside)	1991	INTESTINES	48
+Teit-1908-693-116	116	100	arm	1673	ARM	48
+Teit-1908-693-117	117	101	elbow	981	ELBOW	48
+Teit-1908-693-118	118	102	wrist	799	WRIST	48
+Teit-1908-693-119	119	103	hand	1277	HAND	48
+Teit-1908-693-120	120	104	right hand	2183	RIGHT HAND	48
+Teit-1908-693-121	121	105	left hand	2182	LEFT HAND	48
+Teit-1908-693-122	122	106	palm of hand	1183	PALM OF HAND	48
+Teit-1908-693-123	123	107	back [hand]			50
+Teit-1908-693-124	124	108	finger	1303	FINGER	50
+Teit-1908-693-125	125	109	thumb	1781	THUMB	50
+Teit-1908-693-126	126	110	first finger	577	FOREFINGER	50
+Teit-1908-693-127	127	111	second [finger]	3387	MIDDLE FINGER	50
+Teit-1908-693-128	128	112	third [finger]			50
+Teit-1908-693-129	129	113	small [finger]	2774	LITTLE FINGER	50
+Teit-1908-693-130	130	114	fingernail	1258	FINGERNAIL	50
+Teit-1908-693-131	131	115	leg	1297	LEG	50
+Teit-1908-693-132	132		all leg			50
+Teit-1908-693-133	133	116	upper leg	471	UPPER LEG (THIGH)	50
+Teit-1908-693-134	134	117	knee	1371	KNEE	50
+Teit-1908-693-135	135	118	calf of leg	498	CALF OF LEG	50
+Teit-1908-693-136	136	119	ankle	803	ANKLE	50
+Teit-1908-693-137	137	120	instep	3736	INSTEP	50
+Teit-1908-693-138	138	121	foot	1301	FOOT	50
+Teit-1908-693-139	139	122	sole of foot	2666	SOLE (FOOT)	50
+Teit-1908-693-140	140	123	heel	980	HEEL	50
+Teit-1908-693-141	141	124	toe	1389	TOE	50
+Teit-1908-693-142	142	125	large toe			50
+Teit-1908-693-143	143	126	small toe			50
+Teit-1908-693-144	144	127	toe nail	2662	TOENAIL	50
+Teit-1908-693-145	145	128	rib	801	RIB	50
+Teit-1908-693-146	146	129	blood	946	BLOOD	50
+Teit-1908-693-147	147	130	vein	1924	VEIN	50
+Teit-1908-693-148	148	131	gall	1780	BILE	50
```

Comment:
This is the word for the organ (`GALL BLADDER`, `3372`), this also needs to be fixed in Dawson-1887-731-105, where it is also the organ.

Reply:
How can you tell? In Dawson-1887-731-105 it is explicitly the organ but not here (especially after _blood_). I have nothing against mapping it to GALL BLADDER but just want to understand why.

Reply:
I checked the lexeme and compared it with a (modern) Okanagan dictionary.

Reply:
Then it makes perfect sense, thanks for pointing this out!